klog.go

```diff
@@ -1165,9 +1171,18 @@ func (l *loggingT) createFiles(sev severity) error {
 const flushInterval = 5 * time.Second
 
 // flushDaemon periodically flushes the log file buffers.
-func (l *loggingT) flushDaemon() {
-	for range time.NewTicker(flushInterval).C {
-		l.lockAndFlushAll()
+// added select case to quit channels
+func (l *loggingT) flushDaemon(stop, done chan struct{}) {
+	ticker := time.NewTicker(flushInterval)
+	for {
+		select {
+		case <-stop:
+			continue
```

Comment:
What is the purpose of the `stop` channel here? I originally thought it was to signal a timeout, but the `continue` here is confusing me, shouldn't this return rather than continue?

Reply:
I have added a continue which means that for loop will continue and as soon we get next time ticker on channel It will flush the logs.